### DISCUSSION TITLE
Coordinate System on Napari GUI

### DISCUSSION THREAD

**Discussion by nlasaleta1** [neuropoly/axondeepseg#761] [Help needed]
Hi, 

I have used axondeepseg before, but was familiar with the Fsleyes GUI in the previous version. With the updates and the use of the new Napari GUI I am wondering if Napari has a similar function to Fsleyes where a cross would center on a pair of coordinates you entered. This feature helps me verify if ADS is segmenting properly through manual checks. 

Please let me know when you get a chance. Thanks!

**Comment by hermancollin** [comment]
Hi @nlasaleta1
I don't know about entering a pair of coordinates but maybe this could help you. In the bottom-left corner of the napari window, the coordinates of the cursor are displayed. Using this, you can quickly validate coordinates. Would that be sufficient? Otherwise I can look further, there might be a way to do what you want.
![napari_coord](https://github.com/axondeepseg/axondeepseg/assets/83031821/71965243-a733-4927-9cf7-48a87395385e)
